File: 125-valid-palindrome/valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        String  sl=s.toLowerCase();
        String onlS="";
        for(int i=0;i<sl.length();i++){
            if(sl.charAt(i)>='a' && sl.charAt(i)<='z' || (sl.charAt(i) >= '0' && sl.charAt(i) <= '9')){
                onlS+=sl.charAt(i);
            }
        }
         System.out.println(onlS);
          String rev="";
          for(int i=onlS.length()-1;i>=0;i--){
            rev+=onlS.charAt(i);
          }
           System.out.print(rev);

          return onlS.equals(rev);
    }
}

File: 125-valid-palindrome/valid-palindrome.cs
public class Solution {
    public bool IsPalindrome(string s) {
         string st=s.ToLower();
         string ans=""; 
         foreach(char c in st){
            if((c>='a' && c<='z') || (c>='0' && c<='9'))ans+=c;
         }
         int i=0,j=ans.Length-1;
         while(i<j){
            if(ans[i]!=ans[j])return false;
            i++;j--;
         }
         
         Console.WriteLine(ans);
         return true;
    }
}